lib/database/stats.yml

```diff
@@ -0,0 +1,40 @@
+--- !ruby/object:Game
+hints_used: 0
+attempts_used: 1
+secret_code:
```

Comment:
Why do you store your secret code in such format? Maybe string/integer?
Why do you store arr_for_hints?